File: CreateFile.cpp
#include <iostream>
#include <fstream>
void createfile()
{
	std::string location;
	std::ofstream file;
	file.open(location = "C:\\Users\\User\\ForFileProject\\Log.txt");
	file << "Test";
	std::cout << location;
	std::cin.get();
}
int main()
{
	//call here
	createfile();
}

File: CreateFile.cs
using System;
using System.IO;

class Program
{
    static void Main(string[] args)
    {

        

        string location = "C:\\Users\\User\\CSharpFiles\\Log.txt";

        using (StreamWriter outputFile = new StreamWriter(Path.Combine(location), true))
        {
            outputFile.WriteLine("Text File Created using C#");
        }
    }
}
